website/static/js/registrationUtils.js

```diff
@@ -471,6 +473,12 @@ RegistrationEditor.prototype.init = function(draft) {
     self.currentPage(self.draft().pages()[0]);
 
     self.serialized.subscribe(self.autoSave.bind(self));
+
+    $(window).on('beforeunload', function(e) {
+        if (self.isDirty()) {
```

Comment:
Talked to Andy, we're going to ignore this for now and create a followup ticket.